plugins/attention-and-distributed-packing/src/fms_acceleration_aadp/framework_plugin_padding_free.py

```diff
@@ -71,7 +72,12 @@ def _collator_check(collate_fn):
             # `DataCollatorForSeq2Seq` collate_fn,
             # otherwise the collation can be unreliable"
             return isinstance(
-                collate_fn, (DataCollatorForSeq2Seq, DataCollatorForCompletionOnlyLM)
+                collate_fn,
+                (
+                    DataCollatorForSeq2Seq,
+                    DataCollatorForCompletionOnlyLM,
+                    RemoveColumnsCollator,
+                ),
```

Comment:
Another design here is that if the collator is RemoveColumnsCollator instead of returning True should we extract the underlying collator and then do the check?